joy_teleop/launch/example.launch.py

```diff
@@ -37,11 +37,11 @@ def generate_launch_description():
     ])
 
     ld.add_action(launch_ros.actions.Node(
-            package='joy_teleop', node_executable='joy_teleop',
+            package='joy_teleop', executable='joy_teleop',
             parameters=[launch.substitutions.LaunchConfiguration('teleop_config')]))
 
     ld.add_action(launch_ros.actions.Node(
-            package='joy_teleop', node_executable='incrementer_server',
-            node_name='incrementer', node_namespace='torso_controller'))
```

Comment:
This could be merged into Foxy as-is, if not for #70 , could you please separate the PRs?

Reply:
No problem.